pkg/block/indexheader/binary_reader.go

```diff
@@ -428,11 +426,12 @@ type BinaryReader struct {
 	c io.Closer
 
 	// Map of LabelName to a list of some LabelValues's position in the offset table.
-	// The first and last values for each name are always present.
+	// The first and last values for each name are always present, we keep only `symbolsFactor` part of for rest.
```

Comment:
there is no identifier in code with the name `symbolsFactor` anymore. Maybe we should write this as `1/postingOffsetsInMemSampling` or something

Reply:
good catch! Fixed

Reply:
Good catch, thanks!